src/content/docs/en/reference/container-reference.mdx

```diff
@@ -0,0 +1,292 @@
+---
+title: Astro Container API (experimental)
+i18nReady: false
+---
+import Since from '~/components/Since.astro'
+
+The Container API allows to render Astro components in isolation.
+
+<Since v="4.9.0" />
+
+## `create()`
+
+Creates a new instance of the container. 
+
+```js
+import { experimental_AstroContainer } from "astro/container";
+
+const container = await experimental_AstroContainer.create();
+```
+
+It accepts an object with the following options
+
+
+```ts
+export type AstroContainerOptions = {
+	streaming?: boolean;
+	renderers?: AstroRenderer[];
+	astroConfig?: AstroContainerUserConfig;
+};
+```
+
+#### `streaming` option
+
+Enables rendering components using the streaming.
+
+**Type:** `boolean`
+
+#### `renderers` option
+
+A list of client renderers. Add a renderer if your Astro component renders some client components such as React, Preact, etc.
+
+**Type:**: `AstroRenderer[]`;
+
+For example, if you're rendering a Astro component that renders a React component, you'll have to provide the following renderer: 
```

Comment:
Since we typed the options with `AstroRenderer`, users will be required to provide `name`, a `clientEntrypoint` renderer and `serverEntrypoint` renderer. 

Unfortunately, this is a very low-level API, and no end-user is aware of that other than developers who create Astro renderers. Renders that we support don't expose or document these things. 

Ideally, the example should provide *enough* knowledge as a starting point, but we plan to expose these renderers to the users, so they won't need to manually write them like in the example.

Overall, I think the suggestion looks

Reply:
(NOTE: Part of my thinking by exposing the vocab in a sentence is that even if they can't find these things themselves, it's easy enough for them to come into support and ask, "I can't find the `clientEntrypoint` and `serverEntrypoint` for Svelte..." and then that's an easy question for them to ask and get a direct answer to by someone who knows.)